src/Components/test/E2ETest/Infrastructure/ServerFixtures/ToggleExecutionModeServerFixture.cs

```diff
@@ -41,7 +48,11 @@ protected override string StartAndGetRootUri()
                 underlying.AdditionalArguments.AddRange(AspNetFixtureAdditionalArguments);
                 underlying.BuildWebHostMethod = _buildWebHostMethod;
                 _serverToDispose = underlying;
-                return underlying.RootUri.AbsoluteUri;
+                var uri = underlying.RootUri.AbsoluteUri;
+
+                Host = underlying.Host;
+
+                return uri;
```

Comment:
Same as above

Reply:
Same as above, the server needs to be started so that Host != null